drivers/soundwire/bus.c

```diff
@@ -1258,12 +1258,18 @@ static int sdw_handle_slave_alerts(struct sdw_slave *slave)
 		 * interrupt
 		 */
 		if (buf & SDW_SCP_INT1_PARITY) {
-			dev_err(&slave->dev, "Parity error detected\n");
+			if (slave->prop.scp_int1_mask & SDW_SCP_INT1_PARITY) {
+				dev_err(&slave->dev,
+					"Parity error detected\n");
+			}
```

Comment:
@plbossart Not maybe for this patch, but a general concern that should these errors be of "_ratelimited()" kind? As we don't stop streaming, if a hw fault is hit, this is likely to spam the kernel log at a fairly high rate, right?

Reply:
Nice catch. We did add _ratelimited for the Master side but not for the Slave.
In practice this never happens but regardless we should do this.

Reply:
actually there are a couple of errors that would also fail in case the bus craps out, so we should re-look at the sequence separately, not just add _ratelimited for the parity case only.